app/src/main/java/pm/gnosis/heimdall/ui/addressbook/AddressBookContract.kt

```diff
@@ -13,7 +14,15 @@ import pm.gnosis.svalinn.common.utils.Result
 abstract class AddressBookContract : ViewModel() {
     abstract fun observeAddressBook(): Flowable<Adapter.Data<AddressBookEntry>>
     abstract fun observeAddressBookEntry(address: Solidity.Address): Flowable<AddressBookEntry>
+    abstract fun loadAddressBookEntry(address: Solidity.Address): Single<AddressBookEntry>
     abstract fun addAddressBookEntry(address: String, name: String, description: String = ""): Observable<Result<AddressBookEntry>>
+    abstract fun updateAddressBookEntry(
+        oldAddress: Solidity.Address,
+        name: String,
+        newAddress: String,
```

Comment:
I thought we made the address non-changeable

Reply:
It is changeable